tmt.spec

```diff
@@ -63,6 +64,15 @@ Requires: ansible podman
 All dependencies of the Test Management Tool required to run tests
 in a container environment.
 
+%package testcloud
+Summary: Libvirt (via testcloud) provisioner for the Test Management Tool
+Requires: tmt == %{version}-%{release}
+Requires: ansible testcloud
+
+%description testcloud
+All dependencies of the Test Management Tool required to run tests
+in a libvirt environment provisioned using testcloud.
+
```

Comment:
Include at least `LICENSE`, otherwise the rpm package is not created.

Reply:
ack

Reply:
fixing ...